prereise/gather/demanddata/transportation_electrification/data_helper.py

```diff
@@ -89,3 +92,39 @@ def update_if_weekend(data: pd.DataFrame):
     data.loc[data["Day of Week"].isin(range(2, 7)), "If Weekend"] = 2
 
     return data
+
+
+def generate_daily_weighting(year, area_type="urban"):
+    """Generate daily weighting factors based on vehicle-miles-travelled distributions.
+
+    :param int/str year: year to generate weighting factors for.
+    :param str area_type: Either 'urban' or 'rural'.
```

Comment:
We will need to add the doc string for the error raised here.

Reply:
Done.